### PR TITLE
[readme] Update README.md

### LINKED ISSUE #40: Update Readme (1-2 hours)
update project guide on how to run and set up the project locally and project file structure

### PR DESCRIPTION
**Pull request by ParfenovIgor**
There is no information about migration, as I could not understand how to do it. I could simply run `DataBaseSchema.SQL` to create empty tables.

### CODE REVIEW

**Review by Aardes-Roiven — CHANGES_REQUESTED** [review]
Fix the grammar. There are mistakes

**Comment by ParfenovIgor** [comment]
Recheck. If there are more mistakes, please show them explicitly.

File: README.md [hunk 1]
```diff
@@ -1,99 +1,104 @@
...
 ```
-### Step 4: Run Migrations
-Inside the `LBS` directory run
+
+Here is the expected content of the file:
 ```sh
-poetry run flask db init
-poetry run flask db migrate
-poetry run flask db upgrade
+SECRET_KEY='your_auth_secret'
```
Comment (HisMaj3sty) [inline README.md]: What is this secret?

File: README.md [hunk 2]
```diff
@@ -1,99 +1,104 @@
...
 ```
-### Step 5: Run the application
-Within the `LBS` directory
+
+# Step 4: Define environment variables
 ```sh
-poetry run flask --app app run   
+export POSTGRES_USER=postgres
```
Comment (HisMaj3sty) [inline README.md]: Are you sure these do not belong to the .env file? They are literally *env*ironment variables

File: README.md [hunk 3]
```diff
@@ -1,99 +1,104 @@
...
-Before running, place an .env file in the root directory
+
+### Step 5: Run docker compose
+Run this command when you update the server
 ```sh
 docker compose up --build -d
 ```
-# Project File structure 
+
+### Step 6: Stop docker compose
```
Comment (HisMaj3sty) [inline README.md]: Stopping docker compose is not a part of running the project

File: README.md [hunk 4]
```diff
@@ -1,99 +1,104 @@
...
 ```
-### Step 3: Create .env
 
-The project depends on some config that it expects to be provided from a .env file at the `LBS` directory. 
+### Step 2. Install Docker if it is not installed
+https://docs.docker.com/engine/install/
+
+### Step 3: Create .env file
+
+The project depends on some config that it expects to be provided from an .env file at the `LBS` directory.
```
Comment (HisMaj3sty) [inline README.md]: Is this config equal to the config.py file mentioned in the file structure?

Comment (ParfenovIgor) [reply README.md]: I guess, yes

File: README.md (hunk 1 above)
Comment (ParfenovIgor) [reply README.md]: I don't know. As I understand, it is only required for generating jwt. There is also this field in `.env` file. I don't know, why it is asked to be there.

File: README.md (hunk 2 above)
Comment (ParfenovIgor) [reply README.md]: They are, but they are also required in `docker-compose.yml` to build `DB_URI` environmental variable (it is used in python script to connect to database). Though, I think, we can build it in python directly and don't have these `export`s. I don't know, what is better.

Comment (ParfenovIgor) [reply README.md]: There is actually one more problem with `outside` environmental variables. The docker doesn't look at them and doesn't trigger rebuild image, when they were changed.

File: README.md (hunk 3 above)
Comment (ParfenovIgor) [reply README.md]: Changed

File: README.md (hunk 1 above)
Comment (HisMaj3sty) [reply README.md]: Then learn what it is and write it in the README. It makes 0 sense to write something into README if you have no idea what it is.

**Comment by ParfenovIgor** [comment]
About local installation.
1. It requires local postgres (or any other sql) server, which installation is not simple (it forces us to accurately manipulate with users and their permissions). Should I add instructions on installation of postgres, or assume the user will do it by himself. Also, the DB_URI environmental variable depends on user installation, which may confuse the user.
2. It requires python 3.11, but it is not available on Ubuntu 22 in a straightforward way. Installing `python3.11` will not work, as poetry will still try to use python3.

**Pull request merged**